### PR TITLE
[RORDEV-637] Gradle 7.6.1

### PR DESCRIPTION
**Pull request by mateuszkp96**

### CODE REVIEW

File: ror-tools-core/build.gradle [hunk 1]
```diff
@@ -114,25 +96,21 @@ compileScala {
     options.compilerArgs << "-Ywarn-macros:after"
 }
 
-tasks.withType(ScalaCompile) {
```
Comment (coutoPL) [inline ror-tools-core/build.gradle:117]: maybe this causes the patching step doesn't work in integration tests?

File: tests-utils/build.gradle [hunk 2]
```diff
@@ -48,56 +36,38 @@ tasks.withType(ScalaCompile) {
...
-githook {
-    hooks {
-        "pre-commit" {
-            shell = "./gradlew license --rerun-tasks"
-        }
-    }
 }
 
-task wrapper(type: Wrapper) {
+tasks.register('wrapper', Wrapper) {
```
Comment (coutoPL) [inline tests-utils/build.gradle:71]: is it needed? I don't understand this task. Maybe we can try to remove it? (but please to it after solving all other issues)

**Review by coutoPL — CHANGES_REQUESTED** [review]

**Comment by sscarduzio** [comment]
guys any blocker on this?

**Comment by coutoPL** [comment]
I asked Mateusz to switch to one of the task that our client needs at the moment (see the jira). 

But Mateusz dont hesitate to write a word about problems you have with this one ;)

**Comment by mateuszkp96** [comment]
I resolved problems with all es modules except the proxy module. Gradle has changed dependency management. I need time to find the correct project structure for the proxy module. I think I'm close to a valid solution

**Comment by coutoPL** [comment]
Maybe this is the right time to ask ourselves: do we need the ROR proxy any more?

**Comment by sscarduzio** [comment]
Eventually no user adopted it. If it's an obstacle to moving forward let's drop it. We take it from git if we need it.

File: ror-tools-core/build.gradle (hunk 1 above)
Comment (mateuszkp96) [reply ror-tools-core/build.gradle:117]: Should be fixed now

**Comment by coutoPL** [comment]
yeah, so let's stop supporting the proxy mode. 
@mateuszkp96 could you please get rid of it? If it's better for you, you can do it in a separate PR

**Review by coutoPL — COMMENTED** [review]
It looks good, but I want to play the project with the new gradle in Idea.

**Pull request merged**